cmd/podman/export.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	exportCommand.SetHelpTemplate(HelpTemplate())
 	exportCommand.SetUsageTemplate(UsageTemplate())
 	flags := exportCommand.Flags()
-	flags.StringVarP(&exportCommand.Output, "output", "o", "/dev/stdout", "Write to a file, default is STDOUT")
+	flags.StringVarP(&exportCommand.Output, "output", "o", "/dev/stdout", "Write to a file instead of terminal")
```

Comment:
sorry @edsantiago , I think this should be "instead of the terminal".  IDK I caught that earlier.  WDYT?

Reply:
@TomSweeneyRedHat this is under discussion now, see @rhatdan's [comment](https://github.com/containers/libpod/pull/2633#discussion_r265337021) about "terminal" actually being incorrect. You are good with words, I'd love to hear your suggestion about improved wording (or, if necessary, removing the Cobra default and initializing in code).

Reply:
I'm not a big fan of terminal, when I hear "terminal" I think VT100 Terminal (the real one, not the emulator).  Then afterwards I think of a linux software terminal, so I understand the hesitancy to use the term.

But I think it conveys the fact that the data is going to the screen attached to the process.  Saying STDOUT is closer to what we're doing, but we're also writing info to STDERR.

So I'd stick with "the terminal", but could easily be swayed as I've struggled describing similar circumstances.